docs/README.md

```diff
@@ -4,7 +4,6 @@
 
 This plugin supports a two-way integration between Mattermost and Jira. For a stable production release, please download the latest version [in the Releases tab](https://github.com/mattermost/mattermost-plugin-jira/releases) and follow [these instructions](./#2-configuration) for install and configuration. Check the Plugin Marketplace.
 
-This plugin supports Jira Core and Jira Software products, for Server, Data Center and Cloud platforms. It has been tested with versions 7 and 8.
-
-Support for multiple Jira instances is considered, but not yet supported.
+This plugin supports Jira Core and Jira Software products for Server, Data Center, and Cloud platforms. It has been tested with versions 7 and 8.
 
+**Support for multiple Jira instances is supported from Jira 3.0 and configured using [Administrator Slash Commands](../admininstrator-guide/administrator-slash-commands.md).**
```

Comment:
Github link seems broken.

Reply:
I've replaced the link with the full URL and also listed the installation and configuration pages separately.